src/System.Threading.Tasks.Extensions/src/System/Runtime/CompilerServices/ValueTaskAwaiter.cs

```diff
@@ -0,0 +1,222 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Diagnostics;
+using System.Threading.Tasks;
+using System.Threading.Tasks.Sources;
+
+#if !NETSTANDARD
+using Internal.Runtime.CompilerServices;
+#endif
+
+namespace System.Runtime.CompilerServices
+{
+    /// <summary>Provides an awaiter for a <see cref="ValueTask"/>.</summary>
+    public readonly struct ValueTaskAwaiter : ICriticalNotifyCompletion
+#if CORECLR
```

Comment:
What should I do with the code protected with this directive, both the if and the else?

Reply:
As suggested, removed. It does not show up in ILSpy in any of the DLLs of the package.